README.md

```diff
@@ -107,9 +107,9 @@ See the [Unit test documentation](Source/UnitTests/README.md)
 The current version of Boogie is noted in a [build property](Source/Directory.Build.props).
 To push a new version to nuget, perform the following steps:
 
-- Update the version (e.g., x.y.z) and commit the change
-- git tag vx.y.z
-- git push --tags
+- Update the version (e.g., x.y.z) in `Source/Directory.Build.props` and commit the change locally
+- `git tag vx.y.z`
+- `git push origin vx.y.z` where `origin` denotes the remote on the github.com
```

Comment:
Consider removing `the` before github.com.

Reply:
I think you need to include the master branch in the push command, i.e., `git push origin master vx.y.z`. Otherwise the commit will only be "reachable" via the tag, but others won't see it (and develop on top of it) on the master branch.

Reply:
Ouch. Thanks for the suggestion. (I probably made this mistake when trying to get v2.9.3 out.) I made the edit.